omnibus_overrides.rb

```diff
@@ -10,7 +10,7 @@
 override :ohai, version: "v15.12.0"
 override :ruby, version: "2.6.6"
 override :perl, version: "5.18.1"
-
+override :postgres, version: "12.5"
```

Comment:
this should be postgresql (https://github.com/chef/omnibus-software/compare/lbaker/postgres125-phase1)